src/components/crownConfig.vue

```diff
@@ -89,7 +91,7 @@ export default {
     },
     boundaryEventDropdownData: {
       type: Array,
-      default: null,
+      default: () => [],
```

Comment:
I changed this for consistency since the cog dropdown data is defined in this way.